avocado/core/job.py

```diff
@@ -488,6 +479,23 @@ def _run(self):
 
         return self.exitcode
 
+    def create_test_suite(self):
+        """
+        Creates the test suite for this Job
+
+        This is a public Job API as part of the documented Job phases
```

Comment:
what a "nice" github feature putting the same comment to multiple commits. Very confusing... (not your fault, just beware that I only put one commit and it occurs in different places)